guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -106,45 +157,242 @@ You can customize the version numbers by changing the `major` and `minor` settin
+CONTENT VIEW ID | NAME                      | LABEL                                | COMPOSITE | LAST PUBLISHED      | REPOSITORY IDS
+----------------|---------------------------|--------------------------------------|-----------|---------------------|---------------
+4               | Default Organization View | 6e13cfca-cfb3-4870-9d8b-3bdc0caf97c9 | false     | 2021/03/01 22:50:10 |
+5               | view                      | view                                 | false     |                     |
+----------------|---------------------------|--------------------------------------|-----------|---------------------|---------------
 +
+----
 . To import the Content View version to {ProjectServer}, enter the following command:
 +
 [subs="+quotes"]
 ----
-# hammer content-view version import \
---export-tar /var/lib/pulp/katello-export/_exported_CV_archive_.tar \
---organization-id _Your_Organization_ID_
+# hammer content-import version --content-view=view \
+                                --organization=import-20639 \
+                                --path=/var/lib/pulp/imports/2021-02-25T21-15-22-00-00/
+[............................................................................................................................................................................................................] [100%]
 ----
 +
-Note that until https://bugzilla.redhat.com/show_bug.cgi?id=1745081[BZ#1745081] is resolved, you must enter the full path `/var/lib/pulp/katello-export/`.
-Relative paths do not work.
+Note you must enter the full path `/var/lib/pulp/imports/<path>`. Relative paths do not work.
 +
 . To verify that you import the Content View version successfully, list Content Views for your organization:
 +
 [subs="+quotes"]
 ----
-# hammer content-view version list --organization "_Your_Organization_"
+# hammer content-view version list --content-view=view \
+                                   --organization=import-20639
+---|----------|---------|-------------|-----------------------
+ID | NAME     | VERSION | DESCRIPTION | LIFECYCLE ENVIRONMENTS
+---|----------|---------|-------------|-----------------------
+7  | view 1.0 | 1.0     |             | Library
+---|----------|---------|-------------|-----------------------
+----
+
+
+
+
+[[Using_ISS-Exporting-Library]]
+=== Exporting a Content View Version
+
+You can export contents of all yum repositories in the Library environment of an Organization to an archive file from {ProjectServer} and use this archive file to create the same repositories in another {ProjectServer} or on another {ProjectServer} organization.
+The exported archive file contains the following data:
+
+* A JSON file containing Content View version metadata
+* An archive file containing all the repositories from the Library environment of the Organization.
+
+{ProjectServer} exports only RPM and kickstart files added to a version of a Content View.
```

Comment:
What is this trying to say? 

x exports only the RPM and kickstart files that are included in the Content Vew version? 

Or x exports only the RPM and kickstart files to a version of a Content View?

Reply:
More `x exports only the RPM and kickstart files that are included in the Content Vew version`. Should I just change this to
```suggestion
{ProjectServer} exports only RPM and kickstart files included in a Content View  version.
```

Reply:
LGTM